tests/integration/ha/continuous_writes.py

```diff
@@ -39,13 +39,14 @@ class ContinuousWrites:
     LAST_WRITTEN_VAL_PATH = "last_written_value"
     CERT_PATH = "/tmp/ca_chain.cert"
 
-    def __init__(self, ops_test: OpsTest, app: str):
+    def __init__(self, ops_test: OpsTest, app: str, initial_count: int = 0):
```

Comment:
This shouldn't be needed. As the continuous writes gets deleted at the end of each test, and we start again anew from 0 on new tests.